doc/releases/changelog-dev.md

```diff
@@ -640,6 +640,15 @@
   update the shots and change measurement processes with fewer issues.
   [(#7358)](https://github.com/PennyLaneAI/pennylane/pull/7358)
 
+* Refactored the workflow helper function `construct_batch` to follow the same logic as `qml.execute`.
```

Comment:
```suggestion
* Updated the workflow helper function `construct_batch` to follow the same logic as `qml.execute`.
```

So "refactor" tends to mean a purely structural shift with no change in behaviour, often done to improve maintainability or as prep for further extension.  Here, we are slightly changing behavior, so it's not really a refactor.